hazelcast/src/test/java/com/hazelcast/jmx/MBeanTest.java

```diff
@@ -124,6 +124,15 @@ public void testExecutor() throws Exception {
         holder.assertMBeanExistEventually("IExecutorService", executor.getName());
     }
 
+    @Test
+    public void testReplicatedMap() throws Exception {
```

Comment:
For the scope of this PR, one test that checks the existence of ReplicatedMapMBean is sufficient.
Because the issue which this PR is trying to fix is about existence of ReplicatedMapMBean. If you want to see exhaustive tests for all MBeans, please create a new issue so an available one can pick and fix it.